agent_battler/proxy_manager.py

```
@@ -0,0 +1,172 @@
+
+        # Give it a moment to start
+        await asyncio.sleep(1)
+
+        if self.verbose:
+            print(f"[Proxy] Proxy is running and ready to accept connections")
+    
+    async def _run_proxy(self) -> None:
+        """Run the proxy server."""
+        try:
+            await self.master.run()
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)